prime-router/metadata/fhir_mapping/hl7/datatypes/CNE/CodeableConcept.yml

```diff
@@ -0,0 +1,115 @@
+coding:
+  generateList: true
+  expressionType: nested
+  vars:
+    code: String, CNE.1
+    display: String, CNE.2
+    system: SYSTEM_URL, CNE.3
+    version: String, CNE.7
+  expressionsMap:
+    code:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.1
+    display:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.2
+    system:
+      type: SYSTEM_URL
+      expressionType: HL7Spec
+      valueOf: CNE.3
+    version:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.7
+    extension_CNE-coding:
+      condition: $code NOT_NULL || $display NOT_NULL || $system NOT_NULL || $version NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding
+        valueString:
+          value: coding
+    extension_CNE-coding-system:
+      condition: $coding-system NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        coding-system: String, CNE.3
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding-system
+        valueString:
+          value: $coding-system
+
+coding_alt:
+  generateList: true
+  expressionType: nested
+  vars:
+    altCode: String, CNE.4
+    altDisplay: String, CNE.5
+    altSystem: SYSTEM_URL, CNE.6
+    altVersion: String, CNE.8
+  expressionsMap:
+    code:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.4
+    display:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.5
+    system:
+      type: SYSTEM_URL
+      expressionType: HL7Spec
+      valueOf: CNE.6
+    version:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.8
+    extension_CNE-coding:
+      condition: $altCode NOT_NULL || $altDisplay NOT_NULL || $altSystem NOT_NULL || $altVersion NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding
+        valueString:
+          value: alt-coding
+    extension_CNE-coding-system:
+      condition: $coding-system NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        coding-system: String, CNE.6
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding-system
+        valueString:
+          value: $coding-system
```

Comment:
Oh interesting the NIST spec only goes to CNE.9.  It can't hurt to add them, but I'm going to split out a new ticket that can them and dedicated tests.